From 395d6fca2638129e1cc1f55ee4b3aa0dbba44dc0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20Men=C5=A1=C3=ADk?= <pemensik@redhat.com>
Date: Mon, 16 Jan 2023 11:45:31 +0100
Subject: [PATCH] Include isc_rwlocktype_t type definition in zt.h

After changes zt.h uses rwlock type, which is not enforced to be always
defined. Ensure full type definition is ready by including appropriate
header.
---
 lib/dns/include/dns/zt.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/dns/include/dns/zt.h b/lib/dns/include/dns/zt.h
index 077b26ddea..037d0a9495 100644
--- a/lib/dns/include/dns/zt.h
+++ b/lib/dns/include/dns/zt.h
@@ -18,6 +18,7 @@
 #include <stdbool.h>
 
 #include <isc/lang.h>
+#include <isc/rwlock.h>
 
 #include <dns/types.h>
 
-- 
2.39.0

